modules/acm_sku/src/SKUFieldsManager.php

```diff
@@ -188,29 +187,8 @@ public function updateFieldMetaInfo($field_code, array $field) {
       }
     }
 
-    // Need to apply entity updates for following.
-    $apply_updates = FALSE;
-    $field_labels_info = [
-      'label',
-      'description',
-      'visible_view',
-      'visible_form',
-      'weight',
-    ];
-
-    foreach ($field_labels_info as $info) {
-      if (isset($field[$info]) && $field['type'] != $existing_fields[$field_code]['type']) {
-        $apply_updates = TRUE;
-        break;
-      }
-    }
-
     $existing_fields[$field_code] = array_replace($existing_fields[$field_code], $field);
     $config->setData($existing_fields)->save();
-
-    if ($apply_updates) {
-      $this->entityDefinitionUpdateManager->applyUpdates();
```

Comment:
I don't understand why this code is removed.

Reply:
Automatic entity updates no longer exist in Drupal 8.7+, it is handled in the entity manager for save, therefore the existing code changing the entity should suffice.